acceptance/README.md

```diff
@@ -145,6 +145,33 @@ There also may be scenarios where you want to specify the host(s) to release. E.
     HOST_NAMES=lvwwr9tdplg351u bundle exec rake ci:release_hosts
     HOST_NAMES=lvwwr9tdplg351u,ylrqjh5l6xvym4t bundle exec rake ci:release_hosts
 
+### Running tagged tests
+
+Tests runs can be segmented by tags defined in the tests themselves. These
+tags have been standardized into the following.
+
+* risk:high
+* risk:medium
+* risk:low
```

Comment:
Great questing @glennsarti . I have amended the README to clarify that untagged tests will be run weekly.